spec/workers/rss_reader_fetch_user_worker_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe RssReaderFetchUserWorker, type: :worker do
+  let(:worker) { subject }
+
+  include_examples "#enqueues_on_correct_queue", "medium_priority", [456]
+
+  describe "#perform_now" do
+    let(:rss_reader_service) { instance_double(RssReader) }
+
+    before do
+      allow(RssReader).to receive(:new).and_return(rss_reader_service)
+      allow(rss_reader_service).to receive(:fetch_user)
+    end
+
+    context "when user found and feed_url present" do
+      let(:user) { double }
+
+      before do
+        allow(User).to receive(:find_by).and_return(user)
+        allow(user).to receive(:feed_url).and_return(true)
```

Comment:
I would return smth more similar to the `feed_url` here to avoid confusion.

Reply:
Thanks - updated!